usage/index.adoc

```diff
@@ -74,4 +91,26 @@ If you want to inspect the source code for all of the above it can be
 found
 https://github.com/jbosstools/jbosstools-base/tree/master/usage[here]
 
+== Error Reporting
+
+The error reporting is using https://wiki.eclipse.org/EPP/Logging[Eclipse AERI] which
+been successfully used by Eclipse Foundation to gather error reports from core Eclipse
+plugins.
+
+The error reporting is available when installing most JBoss Tools plugins after JBoss Tools 4.3.2.
+
+When error occurs that looks like it is related to JBoss Tools plugin a notification will pop-up in the lower right outlining the error the user can dismiss, send directly or provide more details.
+
+In the submission you can *optionally* provide an email which we can use to get further information otherwise it is all anonymously.
+
+If you want to fine tune, enable/disable how the error reporting works you can do that in menu:Preferences[General > Error Reporting] that should look similar to this:
+
+image::./images/error-reporting-preferences.png[]
+
+=== How it works
+
+On startup https://tools.jboss.org/usage/errors/rest/2.0/community/discovery is loaded that configures JBoss Tools as an additional receiver of error reports from Eclipse AERI.
+
+It declares which package names and bundles that will be listened to for getting errors most likely related to JBoss Tools.
```

Comment:
This configuration file lists the packages and bundles to which Eclipse will listen for errors, such as org.jboss.* and com.jboss.*.